server/src/main/twirl/views/index.scala.html

```diff
@@ -1,52 +1,45 @@
-    <script async @{CSPNonce.attr} src="https://www.googletagmanager.com/gtag/js?id=@{googleTrackingCode}"></script>
-    <script @{CSPNonce.attr}>
-          window.dataLayer = window.dataLayer || [];
-          function gtag(){dataLayer.push(arguments);};
-          gtag('js', new Date());
-          gtag('config', '@{googleTrackingCode}', {
-              'send_page_view': false,
-          });
-          gtag('consent', 'default', {
-              'ad_storage': 'denied',
-              'ad_user_data': 'denied',
-              'ad_personalization': 'denied',
-              'analytics_storage': 'granted',
-          });
-    </script>
-    }
+  <meta charset="UTF-8">
+  <title>@title</title>
+  <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
+  <link rel="stylesheet" media="screen" href=@_asset("stylesheets/main.min.css")?@BuildInfo.version>
+  <link rel="shortcut icon" type="image/png" href=@_asset("images/favicon.png")>
+  @if(googleTrackingCode.nonEmpty){
+  <!-- Google tag (gtag.js) -->
+  <script async @{CSPNonce.attr} src="https://www.googletagmanager.com/gtag/js?id=@{googleTrackingCode}"></script>
+  <script @{CSPNonce.attr}>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);};
+    gtag('js', new Date());
+    gtag('config', '@{googleTrackingCode}', {
+      'send_page_view': false,
+    });
+    gtag('consent', 'default', {
+      'ad_storage': 'denied',
+      'ad_user_data': 'denied',
+      'ad_personalization': 'denied',
+      'analytics_storage': 'granted',
+    });
+  </script>
+  }
+
+  <script type="module" src="@routes.Assets.versioned("main.js")"></script>
+@*  <script type="module" crossorigin src="/assets/main.js"></script>*@
+@*  <script type="module" crossorigin src="/assets/vendors-node_modules_flickity_js_index_js-library.js"></script>*@
 </head>
 
 <body>
-    <div id="root">
-    </div>
-    <input type="hidden" name="port-code" id="port-code" value="@{port}"/>
-    <input type="hidden" name="ga-code" id="ga-code" value="@{googleTrackingCode}"/>
-    <input type="hidden" name="user-id" id="user-id" value="@{userId}"/>
-    <input type="hidden" name="csrfToken" id="csrfToken" value="@helper.CSRF.getToken.value"/>
+  <div id="root"></div>
+  <input type="hidden" name="port-code" id="port-code" value="@{port}"/>
+  <input type="hidden" name="ga-code" id="ga-code" value="@{googleTrackingCode}"/>
+  <input type="hidden" name="user-id" id="user-id" value="@{userId}"/>
+  <input type="hidden" name="csrfToken" id="csrfToken" value="@helper.CSRF.getToken.value"/>
 </body>
-    @*
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
-      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-    *@
-
-    @Seq("fastopt", "opt").find(optType => env.resource(s"public/client-$optType.js").isDefined).map { appName =>
```

Comment:
would this be still useful if we run build with npm run build ?

Reply:
nope this is old scalajs stuff